guide/payments/hide-sensitive-info.md

```diff
@@ -0,0 +1,107 @@
+
+Pros
+
+- It is quite convenient to switch between revealed and hidden state
+
+Cons
+
+- Super easy for anyone else to reveal your information if they have access to your device.
+
+
+## Entering a pin to unveil information
+
+A problem that seemed to arise throughout each implementation was that is was as easy to reverse the hidden state as it was to enable it. This may be for convenience sake, however, if you are in a situation where you are concerned about unauthorized access by someone who has your device, once hidden perhaps you should be only able to unhide if a PIN or password has been entered. This could therefore reaffirm the identity of the wallet owner for extra security.
+
+![reveal by pin](/assets/images/payments/hide-info-reveal-by-pin.png)
+
+Pros
+- The risk of an unauthorized person revealing your information is minimal due to the pin required
+
+Cons
+
+- It may be annoying having to repeatedly put in your min when ever you want to reveal your information especially if you do so often
+
+  
+
+##  Information hidden by default
+
+Another solution would be to invoke the wallet’s hidden state as a default when the app is opened. What this means is that in the event that someone sees your wallet when you open it, they wouldn't be able to make out the information. The pre-hidden state can be unveiled  after a tap, PIN entry, or perhaps a short 5 second timer.
+
+![hidden by default](/assets/images/payments/hide-info-hidden-by-default.png)
+
+Pros
+
+- Users have some time to assess their environment before their info is displayed.
+
+Cons
+
+- The user may feel a bit frustrated having to wait for their information to be revealed especially in an urgent situation
+
+## Consider removing the hide toggle from the homepage
+
+Having the show/hide button right on the main screen make things quite obvious for someone who has access to your device to press unhide. A solution would be to move the hide toggle away from the home screen and into the app settings. This way, if someone has access to your device and opens the app, they may not immediately know how to reveal the hidden information as it is not as obvious as the previous solutions.
+
+![hide toggle in setting](/assets/images/payments/hide-info-hide-toggle-in-settings.png)
+
+Pros
+- The risk of an unauthorized person revealing your information is minimal due to the fact that the toggle isn't immediately visible on the home screen
+
+Cons
+
+- This solution may not be convenient for a user to quickly hide their information if the need arises.
+
+  
+
+## Why is this pattern important to designers?
+
+Privacy in bitcoin payments goes far beyond hiding balances and other sensitive information, the privacy by design framework states that privacy should be incorporated and built into products by default. This way, whether or not the user is concerned with their data privacy, they would always be protected through good UX.
```

Comment:
This is great paragraph.

Reply:
> through good UX.

Perhaps UI?

Reply:
Do we have a good resource we can link to for "privacy by design framework"? If we mention a framework, we should make it easy to read up on it.

Reply:
This one might be okay - https://www.ipc.on.ca/wp-content/uploads/Resources/7foundationalprinciples.pdf